Foundation/ProcessInfo.swift

```diff
@@ -104,12 +104,25 @@ open class ProcessInfo: NSObject {
         }
         versionString = productVersion._swiftObject
 #elseif os(Windows)
-        var siVersionInfo: OSVERSIONINFOW = OSVERSIONINFOW()
-        siVersionInfo.dwOSVersionInfoSize = DWORD(MemoryLayout<OSVERSIONINFOEXW>.size)
-        if GetVersionExW(&siVersionInfo) == FALSE {
-          return OperatingSystemVersion(majorVersion: fallbackMajor, minorVersion: fallbackMinor, patchVersion: fallbackPatch)
+        let libraryName: [WCHAR] = [ 0x6e, 0x74, 0x6f, 0x73, 0x6b, 0x72, 0x6e, 0x6c, 0x2e, 0x65, 0x78, 0x65, 0x00 ] // ntoskrnl.exe
```

Comment:
This should be from NTDLL.dll, not ntoskrnl.exe.  Writing this as:
```
"ntdll.dll".withCString(as: .utf16) {
  guard let ntdll = LoadLibraryExW($0, nil, DWORD(LOAD_LIBRARY_SEARCH_SYSTEM32)) else {
    ...
  }
  ...
}
```

is much more readable.